### PR TITLE
MLHR-1901 #resolve

### PR DESCRIPTION
**Pull request by tushargosavi**
Changed location of the directory where test files were generated.

### CODE REVIEW

File: library/src/test/java/org/apache/hadoop/io/file/tfile/DTFileTest.java [hunk 1]
```diff
@@ -70,7 +73,7 @@ public void setUp() throws IOException
 
     conf.setInt("tfile.fs.input.buffer.size", tconf.fsInputBufferSize);
     conf.setInt("tfile.fs.output.buffer.size", tconf.fsOutputBufferSize);
-    path = new Path("tmp/dtfile");
+    path = new Path(ROOT, "/tmp/dtfile");
```
Comment (otterc) [inline library/src/test/java/org/apache/hadoop/io/file/tfile/DTFileTest.java]: @tushargosavi we usually create any test data files under target directory

Comment (tushargosavi) [inline library/src/test/java/org/apache/hadoop/io/file/tfile/DTFileTest.java]: @chandnisingh other tests in tfile using same pattern. I used the same path for consistency. This code was copied from apache tfile implementation. Do you want all tests to use target directory for test data?

**Comment by otterc** [comment]
@tushargosavi yes, tests should create files under target directory.

**Comment by tushargosavi** [comment]
@chandnisingh I have updated patch to use target directory for DTFile tests.

**Pull request merged**